### ISSUE TITLE
Process multiple references

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#154]
Implement
```xml
<xref ref="thing-1" ref="thing-2" ref="thing-3">
```
to fundamentally output

```
5.6.23, 7.9.1, 2.3.4
```
along with usual autonaming oprtions, like an autoname.

Original request:  https://groups.google.com/forum/#!topic/mathbook-xml-support/cPI2cwF5POA

**Comment by rbeezer** [comment]
Process `ref`s in turn, adding commas.  Then trim final comma.  Do not implement "and" at the end, let authors use separate `xref` if they want something more complicated.

Wrap in `[]` and `()` as now, else author can split them out.

For autonaming, get name from the first `ref`.  Conceivable, the subsequent `ref` can be checked against the first (via `local-name()` or something better.

**Comment by Alex-Jordan** [comment]
I didn't know you could reuse an attribute like that.
Actually what was more important for my request was `<xrefrange>`, which perhaps could just be `<xref range="yes">`. I know its effect can be done manually with two `<xrefs>` and appropriate auotname attributes. It would just be a convenience.

**Comment by rbeezer** [comment]
On 07/24/2015 11:36 AM, Alex Jordan wrote:
> I didn't know you could reuse an attribute like that.

Well, I think so.  I'll find out.

> Actually what was more important for my request

Patience! It is on my list.  ;-)  Easy stuff first.

**Comment by rbeezer** [comment]
Turns out you cannot reuse an attribute like that. ;-)

So, a list of `xml:id` in a `@ref` gets parsed and dealt with individually.

Implemented at  49eab3f0
